### ISSUE TITLE
size of Mega dataset 

### ISSUE THREAD

**Issue by betterze** [justinpinkney/toonify#5]
Dear Justin,

Thank you for your great work, it is very interesting.

The Mega dataset contains 317 images, which is not a very large size. I just want to clarify, the new model (transfer learning) is trained just using 317 images? How many epoch do you use? Do we need to use low learning rate?

Thank you very much for your help.

Best Wishes,

Alex

**Comment by justinpinkney** [comment]
That's right, the training is only doe for around 36000 images, so for this just a few hundred is enough. This is trained just with the standard StyleGAN2 parameters (not using a lower learning rate or ADA). 

I expect if you try to train longer you will quickly stop seeing improvement as the discriminator will overfit, but for the purposes of starting to adapt to the new image domain this number of images seems to work fine!

**Comment by betterze** [comment]
Thank you for your reply, I understand it now.
